plugins/register-component/src/util/validate.ts

```diff
@@ -19,6 +19,6 @@ export const ComponentIdValidators = {
     (typeof value === 'string' && value.match(/^https:\/\//) !== null) ||
     'Must start with https://.',
   yamlValidator: (value: any) =>
-    (typeof value === 'string' && value.match(/.yaml$/) !== null) ||
-    "Must end with '.yaml'.",
+    (typeof value === 'string' && value.match(/.yaml/) !== null) ||
```

Comment:
i see that the dot actually needs a backslash escape too - otherwise `_yaml` matches too :) would you mind adding that while you're there?